Reconstruction/RecCalorimeter/tests/options/geant_fullsim_combinedCalo_singleparticles.py

```diff
@@ -0,0 +1,168 @@
+
+# range cut                                                                                                                                                                                                                        
+geantservice.G4commands += ["/run/setCut 0.1 mm"]
+
+# Magnetic field                                                                                                                                                                                                                           
+from Configurables import SimG4ConstantMagneticFieldTool
+field = SimG4ConstantMagneticFieldTool("SimG4ConstantMagneticFieldTool",FieldOn=False)
+
+# Geant4 algorithm                                                                                                                                                                                                       
+# Translates EDM to G4Event, passes the event to G4, writes out outputs via tools                                                                                                                                          
+# and a tool that saves the calorimeter hits                                                                                                                                                                                         
+from Configurables import SimG4Alg, SimG4SaveCalHits, InspectHitsCollectionsTool
+saveecaltool = SimG4SaveCalHits("saveECalHits", readoutNames = ["ECalHitsEta"],
+                                positionedCaloHits = "ECalPositionedHits",
+                                caloHits = "ECalHits")
+savehcaltool = SimG4SaveCalHits("saveHCalHits",readoutNames = ["BarHCal_Readout"],
+                                positionedCaloHits="HCalPositionedHits",
+                                caloHits="HCalHits")
+
+# next, create the G4 algorithm, giving the list of names of tools ("XX/YY")                                                                                                                                                     
+from Configurables import SimG4SingleParticleGeneratorTool
+pgun = SimG4SingleParticleGeneratorTool("SimG4SingleParticleGeneratorTool",saveEdm=True,
+                particleName=particleType,energyMin=energy,energyMax=energy,etaMin=0.36,etaMax=0.36,
+                OutputLevel =DEBUG)
+
+geantsim = SimG4Alg("SimG4Alg",
+                       outputs= ["SimG4SaveCalHits/saveECalHits", "SimG4SaveCalHits/saveHCalHits"],
+                       eventProvider=pgun,
+                       OutputLevel=DEBUG)
+
+# common ECAL specific information
+# readout name
+ecalReadoutName = "ECalHitsEta"
+# active material identifier name
+ecalIdentifierName = "active_layer"
+# active material volume name
+ecalVolumeName = "LAr_sensitive"
+# ECAL bitfield names & values
+ecalFieldNames=["system","ECAL_Cryo","bath","EM_barrel"]
+ecalFieldValues=[5,1,1,1]
```

Comment:
Hi Coralie, that actually slipped by me. We usually try and remove code that is not used (including unused variables), to avoid having deprecated / wrong values around the code that lead to confusion later. So I vote to remove the lines and re-add them, once they are indeed needed. 
Thanks, Joschka